src/Components/Element.php

```diff
@@ -296,6 +296,15 @@ final public function snakecase(): static
         return new static($this->output, $value, $this->properties);
     }
 
+    /**
+     * Adds a hyperlink to the element.
+     */
+    final public function href(string $href): static
+    {
+        // @phpstan-ignore-next-line
```

Comment:
why ignore ?

Reply:
The `properties` parameter is `array<string, array<int|string, int|string>>` but this is an `array<string, string>`.